cmd/relay-gossip/main.go

```diff
@@ -142,14 +138,14 @@ var runCmd = &cli.Command{
 		for {
 			conn, err := grpc.Dial(cctx.String("connect"), opts...)
 			if err != nil {
-				log.Warnf("error connecting to grpc: %+v", err)
+				log.Warn("error connecting to grpc:", err)
```

Comment:
the `dlog` is expecting pairs of arguments that become key/value pairs
```suggestion
				log.Warn("relay-gossip","error connecting to grpc", "err", err)
```

Reply:
oh I see, thanks!